manager/controllers/app/plotter_generator.go

```diff
@@ -152,10 +152,10 @@ func (p *PlotterGenerator) AddFlowInfoForAsset(item DataInfo, appContext *app.Fy
 	datasetID := item.Context.DataSetID
 	for _, element := range selection.DataPath {
 		moduleCapability := element.Module.Spec.Capabilities[element.CapabilityIndex]
-		p.Log.Trace().Msg("Adding module for " + moduleCapability.Capability)
+		p.Log.Trace().Msgf("Adding module for %s", string(moduleCapability.Capability))
```

Comment:
I don't think that you need string() now

Reply:
ok